Methods.h

```diff
@@ -0,0 +1,50 @@
+#include "Classes.h"
+using namespace std;
+
+void Smartphone::add(float p, string n, float M, float A, float B)
+{
+    name = n;
+    price = p;
+    Memory_capacity = M;
+    Array_of_mobile_numbers = A;
+    Battery_capacity = B;
+}
+void Smartphone::out()
+{
+    cout << "Name: ";
+    cout << name << endl;
+    cout << "Price: ";
+    cout << price << endl;
+    cout << "Memory_capacity: ";
+    cout << Memory_capacity << endl;
+    cout << "Array_of_mobile_numbers: ";
+    cout << Array_of_mobile_numbers << endl;
+    cout << "Battery_capacity: ";
+    cout << Battery_capacity << endl << endl;
+}
+
+void Shop::best(float price) {
```

Comment:
You are using unclear names for the class methods.

You should:
 void SmartphoneShop::print_BestSmartphoneByPrice(float max_price) {